be/src/runtime/load_channel.h

```diff
@@ -64,7 +64,7 @@ class LoadChannel {
     // The method will not return until the chosen tablet channels finished memtable
     // flush.
     template <typename TabletWriterAddResult>
-    Status handle_mem_exceed_limit(TabletWriterAddResult* response);
+    void handle_mem_exceed_limit(TabletWriterAddResult* response);
```

Comment:
remove the parameter `response` as well, it's useless now

Reply:
done.